generator_hdl.py

```diff
@@ -323,6 +323,7 @@ def getMemPrintDirectory(fname):
                         help="Detector region. A: all, L: barrel, D: disk")
     
     parser.add_argument('--uut', type=str, default=None, help="Unit Under Test")
+    parser.add_argument('--mut', type=str, choices=["IR","VMR", "TE", "TC", "PR", "ME", "MC"], default="PR", help="Module Under Test")
```

Comment:
I think the default here should be None, and we should allow None as a possible option as well.